hw/ip/csrng/data/csrng.hjson

```diff
@@ -0,0 +1,165 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "csrng",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  param_list: [
+    { name: "NHwApps",
+      type: "int",
+      default: "4",
+      desc: "Number of CSRNG hardware applications",
+      local: "true"
+    } 
+  ],
+  interrupt_list: [
+    { name: "cs_cmd_req_done"
+      desc: "Asserted when a command request is completed."}
+    { name: "cs_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ],
+  inter_signal_list: [
```

Comment:
I think `entropy_src_hw_if` is not defined here but used in csrng.

Also, `efuse_sw_app_enable signal isn't defined too.